src/languages/typescript.js

```diff
@@ -69,19 +69,22 @@ export default function(hljs) {
     "satisfies"
   ];
 
-  /*
-    namespace is a TS keyword but it's fine to use it as a variable name too.
-    const message = 'foo';
-    const namespace = 'bar';
-  */
-
+  const OPTIONAL_PROPERTY_RE = IDENT_RE + '\\s*\\?';
```

Comment:
Please provide an example of what this is trying to do.

Reply:
in my new commit ive added some test so i found that it was failing on some new tcs so ill explain the new one